### PR TITLE
add oscp 4.18 osci configs for jboss eap

### PR DESCRIPTION
**Pull request by amp-rh**
add the following jobs for ocp 4.18:
  - eap-74-interop-aws
  - eap-74-interop-aws-fips
  - eap-74-xp-interop-aws
  - eap-74-xp-interop-aws-fips

### CODE REVIEW

**Comment by amp-rh** [comment]
/pj-rehearse pull-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.18-lp-interop-images

File: ci-operator/config/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x__eap-ocp4.18-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        ]
+      USER_TAGS: |
+        scenario eap
+    test:
+    - ref: install-operators
+    - ref: eap-pit-74-execute-tests
+    - ref: eap-xp-execute-tests
+    workflow: firewatch-ipi-aws
+- as: eap-74-xp-interop-aws-fips
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x__eap-ocp4.18-lp-interop.yaml:119]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/config/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x__eap-ocp4.18-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            {"name": "eap", "source": "redhat-operators", "channel": "stable", "version": "2.4.0", "install_namespace": "openshift-operators"}
+        ]
+      USER_TAGS: |
+        scenario eap
+    test:
+    - ref: install-operators
+    - ref: eap-pit-74-execute-tests
+    workflow: firewatch-ipi-aws
+- as: eap-74-xp-interop-aws
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x__eap-ocp4.18-lp-interop.yaml:87]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/config/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x__eap-ocp4.18-lp-interop.yaml [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: eap-74-interop-aws
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x__eap-ocp4.18-lp-interop.yaml:28]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/config/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x__eap-ocp4.18-lp-interop.yaml [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+            {"name": "eap", "source": "redhat-operators", "channel": "stable", "version": "2.4.0", "install_namespace": "openshift-operators"}
+        ]
+      USER_TAGS: |
+        scenario eap
+    test:
+    - ref: install-operators
+    - ref: eap-pit-74-execute-tests
+    workflow: firewatch-ipi-aws
+- as: eap-74-interop-aws-fips
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x__eap-ocp4.18-lp-interop.yaml:57]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 5]
```diff
@@ -983,3 +972,399 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:977]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 6]
```diff
@@ -983,3 +972,399 @@ periodics:
...
+        secretName: manifest-tool-local-pusher
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:1076]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 7]
```diff
@@ -983,3 +972,399 @@ periodics:
...
```
Comment (dfrazzette) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:1175]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 8]
```diff
@@ -983,3 +972,399 @@ periodics:
...
```
Comment (dfrazzette) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:1274]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 9]
```diff
@@ -16,17 +16,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.14-lp-interop-eap-74-interop-ibmcloud
```
Comment (dfrazzette) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:18]:
```suggestion
  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.14-lp-interop-eap-74-interop-ibmcloud
  reporter_config:
    slack:
      channel: '#eap-ocp-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :success: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :dumpster-fire:
        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> {{end}}'
```

**Pull request merged**